locale/it/site.json

```diff
@@ -0,0 +1,135 @@
+{
+    "title": "Node.js",
+    "author": "Node.js Foundation",
+    "url": "https://nodejs.org/en/",
+    "locale": "it",
+    "scrollToTop": "Portami all\'inizio",
+    "reportNodeIssue": "Segnala un problema di Node.js",
+    "reportWebsiteIssue": "Segnala un problema del sito",
+    "by": "da",
+    "all-downloads": "Tutti i download",
+    "feeds": [
+        {
+            "link": "feed/blog.xml",
+            "text": "Blog Node.js"
+        },
+        {
+            "link": "feed/releases.xml",
+            "text": "Blog Node.js: Releases"
+        },
+        {
+            "link": "feed/vulnerability.xml",
+            "text": "Blog Node.js: Rapporti di Vulnerabilità"
+        },
+        {
+            "link": "feed/tsc-minutes.xml",
+            "text": "Minute Incontri TSC Node.js"
+        }
+    ],
+    "home": { "text": "Home" },
+    "about": {
+        "link": "about",
+        "text": "About",
+        "governance": {
+            "link": "about/governance",
+            "text": "Governance"
+        },
+        "workinggroups": {
+            "link": "about/working-groups",
+            "text": "Gruppi di lavoro"
+        },
+        "releases": {
+            "link": "about/releases",
+            "text": "Releases"
+        },
+        "resources": {
+            "link": "about/resources",
+            "text": "Risorse"
+        }
+    },
+    "download": {
+        "link": "download",
+        "text": "Scarica",
```

Comment:
For consistency I would use "Download" here. This is the only place where "Scarica" is used.